debian/ubuntu-advantage-tools.postinst

```diff
@@ -256,11 +256,11 @@ remove_old_systemd_units() {
             # The failed state is ephemeral and only needs to be cleared if
             # it is there so that the system doesn't say it is degraded.
             # If the old timer was not running, then this is a noop.
-            systemctl --system daemon-reload > /dev/null || true
-            systemctl reset-failed ua-license-check.timer > /dev/null 2>&1 || true
+            deb-systemd-invoke --system daemon-reload > /dev/null || true
+            deb-systemd-invoke reset-failed ua-license-check.timer > /dev/null 2>&1 || true
             # In rare race-condition scenarios, the service can also get into
             # the same failed state.
-            systemctl reset-failed ua-license-check.service > /dev/null 2>&1 || true
+            deb-systemd-invoke reset-failed ua-license-check.service > /dev/null 2>&1 || true
```

Comment:
`deb-systemd-invoke` only supports start stop and restart which is why we have to use systemctl here
See https://salsa.debian.org/debian/init-system-helpers/-/blob/master/script/deb-systemd-invoke

Reply:
Aha! yep my grepping failed me there. Cool +1 to this change then; I think we need a changelog entry to go with it though?

Reply:
Good idea, +1 to having a changelog entry (I'm all for having more changelog entries!).